substrate/frame/multisig/src/lib.rs

```diff
@@ -240,15 +263,19 @@ pub mod pallet {
 		},
 		/// A multisig operation has been cancelled.
 		MultisigCancelled {
-			cancelling: T::AccountId,
```

Comment:
any specific reason this was removed?

Reply:
The reason for this is explained [here](https://github.com/paritytech/substrate/pull/14444#discussion_r1272619034) but as you mentioned I think both me and Shawn misinterpreted the `cancel_as_multi` extrinsic. 
My understanding of it at least was that with this extrinsic the members of the multisig can collectively cancel an operation, but this is not how the extrinsic actually works.

Reply:
I will revert this change.